youtokentome/cpp/bpe.cpp

```diff
@@ -1862,7 +1869,8 @@ Status BaseEncoder::decode(const vector<string> &data, vector<string> *sentences
       ids.push_back(x);
     }
     string sentence;
-    Status status = decode(ids, &sentence);
+    unordered_set<int> empty_ignore;
```

Comment:
that was @yutkin idea

Reply:
Actually, because you could not pass a pointer. Of course, the `nullptr` is better.